### PR TITLE
Spring Boot integration

### PR DESCRIPTION
**Pull request by egor-geller**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -12,10 +12,16 @@ allprojects {
         implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: jacksonVersion
         implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: jacksonVersion
         implementation group: 'org.apache.commons', name: 'commons-lang3', version: apacheCommon
+
+        //Dependency to converter Duration to string and vice versa
```
Comment (nickaden) [inline build.gradle]: I'm not sure that hibernate-core is for only Duration conversion :)

File: build.gradle [hunk 2]
```diff
@@ -12,10 +12,16 @@ allprojects {
...
         implementation group: 'org.apache.commons', name: 'commons-lang3', version: apacheCommon
+
+        //Dependency to converter Duration to string and vice versa
         implementation group: 'org.hibernate', name: 'hibernate-core', version: hibernateVersion
 
-        testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junitVersion
-        testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junitVersion
+        implementation group: 'org.springframework.boot', name: 'spring-boot-starter', version: bootVersion
+        implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: bootVersion
+        implementation group: 'org.springframework.boot', name: 'spring-boot-starter-hateoas', version: bootVersion
```
Comment (nickaden) [inline build.gradle]: Could you please move these dependencies to `web` module?

File: build.gradle (hunk 1 above)
Comment (egor-geller) [reply build.gradle]: https://github.com/egor-geller/certificate/pull/6#issue-1042064916

File: build.gradle (hunk 2 above)
Comment (egor-geller) [reply build.gradle]: https://github.com/egor-geller/certificate/pull/6#discussion_r741859927

**Pull request merged**